src/SFA.DAS.AANHub.Application/CalendarEvents/Commands/CreateCalendarEvent/CreateCalendarEventCommandValidator.cs

```diff
@@ -0,0 +1,246 @@
+    public const string ContactEmailMustBeValid = "contactEmail must be a valid email format";
+    public const string PlannedAttendeesMustNotBeEmpty = "plannedAttendees must have a value";
+    public const string PlannedAttendeesMustBeValid = "plannedAttendees must be  between 1 to 1000000";
+    public const string RequestedByMemberIdMustNotBeEmpty = "requestedByMemberId must have a value";
+    public const string RequestedByMemberIdMustBeAdmin = "requestedByMemberId must be an active, admin member or regional chair";
+
+    public CreateCalendarEventCommandValidator(
+        ICalendarsReadRepository calendarsReadRepository,
+        IRegionsReadRepository regionsReadRepository,
+        IMembersReadRepository membersReadRepository)
+    {
+        RuleLevelCascadeMode = CascadeMode.Stop;
+
+        RuleFor(c => c.AdminMemberId)
+            .NotEmpty()
+            .WithMessage(RequestedByMemberIdMustNotBeEmpty)
+            .MustAsync(async (memberId, cancellationToken) =>
+            {
+                var member = await membersReadRepository.GetMember(memberId);
+                return
+                    member != null &&
+                    member!.Status == MembershipStatusType.Live.ToString() &&
+                    (member.UserType == UserType.Admin.ToString() || member.IsRegionalChair.GetValueOrDefault());
+            })
+            .WithMessage(RequestedByMemberIdMustBeAdmin)
+            .DependentRules(() =>
+            {
+                RuleFor(c => c.CalendarId)
+                    .GreaterThan(0)
+                    .WithMessage(CalendarTypeIdMustNotBeEmpty)
+                    .MustAsync(async (id, cancellationToken) =>
+                    {
+                        var calendarTypes = await calendarsReadRepository.GetAllCalendars(cancellationToken);
+                        return calendarTypes.Exists(c => c.Id == id);
+                    })
+                    .WithMessage(CalendarTypeIdMustBeValid);
+
+                RuleFor(c => c.EventFormat)
+                    .NotEmpty()
+                    .WithMessage(EventFormatMustNotBeEmpty);
+
+                RuleFor(c => c.StartDate)
+                    .Cascade(CascadeMode.Stop)
+                    .NotEmpty()
+                    .WithMessage(StartDateMustNotBeEmpty)
+                    .GreaterThan(DateTime.UtcNow)
+                    .WithMessage(StartDateMustBeInFuture)
+                    .LessThanOrEqualTo(c => c.EndDate)
+                    .WithMessage(StartDateMustBeLessThanEndDate);
+
+                RuleFor(c => c.EndDate)
+                    .NotEmpty()
+                    .WithMessage(EndDateMustNotBeEmpty)
+                    .GreaterThan(DateTime.UtcNow)
+                    .WithMessage(EndDateMustBeInFuture)
+                    .GreaterThanOrEqualTo(c => c.StartDate)
+                    .WithMessage(EndDateMustBeLessThanEndDate);
```

Comment:
You don't need two messages about start date falling after enddate.  I suggest (as it matches the FE validation messages) remove the check from the startdate, and leave the enddate one in.